pkg/workspace/project.go

```diff
@@ -4,24 +4,44 @@
 package workspace
 
 import (
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
 	"fmt"
 	"strings"
 
 	"github.com/daytonaio/daytona/internal"
 	"github.com/daytonaio/daytona/pkg/gitprovider"
 )
 
+type ProjectBuildDevcontainer struct {
+	DevContainerFilePath string `json:"devContainerFilePath"`
+} // @name ProjectBuildDevcontainer
+
+type ProjectBuildDockerfile struct {
+	Context    string            `json:"context"`
+	Dockerfile string            `json:"dockerfile"`
+	Args       map[string]string `json:"args"`
+} // @name ProjectBuildDockerfile
+
+type ProjectBuild struct {
+	Devcontainer *ProjectBuildDevcontainer `json:"devcontainer"`
+	Dockerfile   *ProjectBuildDockerfile   `json:"dockerfile"`
```

Comment:
I suggest we comment this out since we don't support it yet.

Reply:
I have commented out the Dockerfile, but I will leave it there as it will be done right after.